src/current/v23.2/physical-cluster-replication-overview.md

```diff
@@ -0,0 +1,132 @@
+- **Cutover to a timestamp in the past**: In the case of logical disasters or mistakes, you can cut over from the primary to the standby cluster to a timestamp in the past. This means that you can return the standby to a timestamp before the mistake was replicated to the standby. You can also configure the `WITH RETENTION` option to control how far in the past you can cut over to. {% comment %}link retention option here{% endcomment %}
+- **Monitoring**: To monitor the replication progress, you can use metrics available in the [DB Console]({% link {{ page.version.version }}/ui-overview.md %}) and [Prometheus]({% link {{ page.version.version }}/monitor-cockroachdb-with-prometheus.md %}). For more detail, refer to [Physical Cluster Replication Monitoring]({% link {{ page.version.version }}/physical-cluster-replication-monitoring.md %}).
+
+{% comment %}-**Data verification on standby**: You can verify that the data on the standby cluster matches that on the primary by checking the fingerprint of the data on the primary to compare to the standby. We recommend running data verification checks regularly as part of your monitoring processes. Refer to the [Physical Cluster Replication Monitoring]({% link {{ page.version.version }}/physical-cluster-replication-monitoring.md %}) page for a guide and considerations on data verification.{% endcomment %}
+
+{{site.data.alerts.callout_info}}
+[Cutting over to a timestamp in the past]({% link {{ page.version.version }}/cutover-replication.md %}#cut-over-to-a-point-in-time) involves reverting data on the standby cluster. As a result, the time it takes to cut over will be longer than cutover to the latest replicated time. The increase in cutover time will correlate to how much data you are reverting from the standby. For more detail, refer to the [Technical Overview]({% link {{ page.version.version }}/physical-cluster-replication-technical-overview.md %}) page for physical cluster replication.
+{{site.data.alerts.end}}
+
+## Limitations
+
+- Physical cluster replication is only supported on CockroachDB {{ site.data.products.core }} in new v23.2 clusters. That is, clusters that have been upgraded from a previous version of CockroachDB will not support physical cluster replication.
+- Read queries are not supported on the standby cluster before cutover. {% comment %}To verify the data, use new fingerprint feature{% endcomment %}
+- The primary and standby cluster **cannot have different [region topology]({% link {{ page.version.version }}/topology-patterns.md %})**. For example, a single-region primary cluster to a multi-region standby cluster. Or, mismatching regions between a multi-region primary and standby cluster.
```

Comment:
```suggestion
- The primary and standby cluster **cannot have different [region topology]({% link {{ page.version.version }}/topology-patterns.md %})**. For example, a replicating a single-region primary cluster to a multi-region standby cluster is not supported. Mismatching regions between a multi-region primary and standby cluster is also not supported.
```
This makes me more confused about https://github.com/cockroachdb/docs/pull/17989/files#diff-095ec7fe48d1abddb2f67cca5447102d022504028606bc1b93bb4f3d102cfc55R24.

Reply:
I hope this is clearer with the clarification added earlier....